tests/system/providers/amazon/aws/example_lambda.py

```diff
@@ -103,7 +103,7 @@ def delete_lambda(function_name: str):
     invoke_lambda_function = LambdaInvokeFunctionOperator(
         task_id="invoke_lambda_function",
         function_name=lambda_function_name,
-        payload=json.dumps({"SampleEvent": {"SampleData": {"Name": "XYZ", "DoB": "1993-01-01"}}}),
+        payload=json.dumps({"SampleEvent": {"SampleData": {"Name": "XYZ", "DoB": "1993-01-01"}}}).encode(),
```

Comment:
I wonder if this change is necessary. Do we want to encourage airflow users to send bytes (which is consistent with boto3) or leave it in the current way of sending str?

Reply:
I would leave it as str. The operator/hook takes care of the encoding